rapids/test_rapids.py

```diff
@@ -47,24 +44,29 @@ def verify_spark_job(self):
           "{}/rapids/{}".format(self.INIT_ACTIONS_REPO,
                                 self.XGBOOST_SPARK_TEST_SCRIPT_FILE_NAME))
 
-  @parameterized.parameters(("STANDARD", ["m", "w-0"], GPU_P100))
-  def test_rapids_dask(self, configuration, machine_suffixes, accelerator):
-    if self.getImageVersion() < pkg_resources.parse_version("1.5"):
-      self.skipTest("Not supported in pre 1.5 images")
-
-    optional_components = None
+  @parameterized.parameters(
+    ("STANDARD", ["m", "w-0"], GPU_P100, None),
+    ("STANDARD", ["m", "w-0"], GPU_P100, "yarn"),
+    ("STANDARD", ["m"], GPU_P100, "standalone"))
+  def test_rapids_dask(self, configuration, machine_suffixes, accelerator, dask_runtime):
     if self.getImageVersion() < pkg_resources.parse_version("2.0"):
-      optional_components = ["ANACONDA"]
+      self.skipTest("Not supported in pre 2.0 images")
+
+    init_actions = self.INIT_ACTIONS[:]
+    init_actions.insert(1, "dask/dask.sh")
```

Comment:
Maybe better to define `DASK_INIT_ACTIONS` constant, instead of inserting init actions in existing `INIT_ACTIONS` array - test should be more robust this way.

Reply:
done